Shared/Extensions/KeychainWrapperExtensions.swift

```diff
@@ -5,13 +5,11 @@
 import Foundation
 import SwiftKeychainWrapper
 
-let baseBundleIdentifier = AppInfo.baseBundleIdentifier()!
-let accessGroupPrefix = Bundle.main.object(forInfoDictionaryKey: "AppIdentifierPrefix") as! String
-let accessGroupIdentifier = AppInfo.keychainAccessGroupWithPrefix(accessGroupPrefix)!
-private var sharedAppKeychainWrapper = KeychainWrapper(serviceName: baseBundleIdentifier, accessGroup: accessGroupIdentifier)
-
 public extension KeychainWrapper {
     static var sharedAppContainerKeychain: KeychainWrapper {
-        return sharedAppKeychainWrapper
+        let baseBundleIdentifier = AppInfo.baseBundleIdentifier()!
```

Comment:
Having this inside the method should be the same as having it out since `static` vars are the new Swift 3 singletons.